src/aro/TypeStore.zig

```diff
@@ -1642,6 +1646,7 @@ pub const Type = union(enum) {
     pub const Pointer = struct {
         child: QualType,
         decayed: ?QualType,
+        bounds: Attribute.PointerBounds,
```

Comment:
Could you make this default to `.c`? Probably could be applied to `decayed` as well since these are both rarely set to anything else.